src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/NoWhitespaceAfterCheck.java

```diff
@@ -91,14 +91,111 @@
  * <pre>
  * &lt;module name=&quot;NoWhitespaceAfter&quot;/&gt;
  * </pre>
- * <p>To configure the check to forbid linebreaks after a DOT token:
+ *
+ * <p>
+ * Example:
+ * </p>
+ * <pre>
+ * class Test2 {
+ *   public void function1() {}
+ *   public void function2() {}
+ * }
+ *
+ * class Test extends Test2 {
```

Comment:
Please remove Test2
And override from https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/lang/Object.html
toString() and finalize()

Reply:
I do not see it done